.gitignore

```diff
@@ -4,6 +4,7 @@ browsers
 coverage
 dist
 node_modules
+test/lib/sdkdriver/lib
```

Comment:
Additionally, we may want to add this to `.npmignore`, since we probably don't want to publish the TypeScript files. (Or is `test/` already in `.npmignore`?).

Reply:
@markandrus `test/` is already in `.npmignore`.